apps/web/app/api/v1/client/displays/[displayId]/responded/route.ts

```diff
@@ -14,14 +10,17 @@ export async function OPTIONS(): Promise<NextResponse> {
 export async function POST(_: Request, { params }: { params: { displayId: string } }): Promise<NextResponse> {
   const { displayId } = params;
 
-  const display = await prisma.display.update({
-    where: {
-      id: displayId,
-    },
-    data: {
-      status: "responded",
-    },
-  });
+  if (!displayId) {
+    return responses.badRequestResponse("Missing displayId");
+  }
+
+  let display: TDisplay;
```

Comment:
Instead of having this line I would move the response return
```
  return responses.successResponse(
    {
      ...display,
      createdAt: display.createdAt.toISOString(),
      updatedAt: display.updatedAt.toISOString(),
    },
    true
  );
  ```
 inside the try block